File: Sample/src/main/java/ir/houmansanati/persiantimeago/MainActivity.java
package ir.houmansanati.persiantimeago;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import ir.houmansanati.perisantimeago.PersianTimeAgo;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        String timeago = PersianTimeAgo.Companion.getTimeAgo("2019-01-25 02:32:58");
    }
}

File: Sample/src/main/java/ir/houmansanati/persiantimeago/MainActivity.kt
package ir.houmansanati.persiantimeago

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView

class MainActivity : AppCompatActivity() {
    lateinit var recyclerView: RecyclerView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //setup recyclerview
        recyclerView = findViewById(R.id.recyclerview)
        val adapter = TimeRecyclerviewAdapter(this, FakeTimeGenerator.getTimes())
        recyclerView.layoutManager = LinearLayoutManager(this, RecyclerView.VERTICAL, false)
        recyclerView.adapter = adapter
    }
}
